mewarpx/examples/thermionic_diode_rz_cylinder.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+import sys
+
+from mewarpx.utils_store import util as mwxutil
+
+mwxutil.init_libwarpx(ndim=2, rz=True)
+
+import numpy as np
+from pywarpx import picmi
+
+from mewarpx import assemblies, diags, emission, mespecies, runinfo
+from mewarpx.mwxrun import mwxrun
+
+
+class VacuumTEC_Cylinder(object):
+
+    #######################################################################
+    # Begin global user parameters                                        #
+    #######################################################################
+
+    CATHODE_TEMP = 1100 + 273.15 # K
+    CATHODE_A = 6e5              # A/m^2/K^2
+    CATHODE_PHI = 2.11           # eV
+    USE_SCHOTTKY = False
+    V_CATHODE = -CATHODE_PHI     # cathode is grounded
+
+    ANODE_TEMP = 200             # K
+    ANODE_PHI = 1.4              # eV
+
+    D_CA = 50e-6                 # m
+
+    NZ = 32
+    NR = 512
+
+    DT = 0.5e-12                 # s
```

Comment:
What voltage does this satisfy CFL up to? This seems like a place a user could easily confuse themselves with bad results.

Reply:
Not sure, but it looks like it is properly resolved up to 8 V where the full J_sat is already pulled out. So if someone changes the cathode temperature or WF they should also confirm that the simulation is still resolved (also with regards to the spatial resolution).